static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -79,6 +78,25 @@ export function InviteBanner({
     if (!isEligibleForBanner) {
       return;
     }
+    async function fetchMissingMembers() {
+      try {
+        const data = await api.requestPromise(
+          `/organizations/${organization.slug}/missing-members/`,
+          {
+            method: 'GET',
+          }
+        );
+        const githubMissingMembers = data?.filter(
+          integrationMissingMembers => integrationMissingMembers.integration === 'github'
+        )[0];
+        setMissingMembers(githubMissingMembers?.users);
+      } catch (err) {
+        if (err.status !== 403) {
+          addErrorMessage(t('Unable to fetching missing commit authors'));
+        }
+      }
+    }
+    fetchMissingMembers();
```

Comment:
you've defined this as an async method - but not awaiting it here on call?
i think maybe you want to turn this into a useCallback instead?

Reply:
i think useCallback is a good call